index.js

```diff
@@ -153,6 +175,33 @@ function render({ feed, meta, attr }, info, res) {
     .catch(e => sendError(res, e));
 }
 
+/**
+ * Finds the index of a given element in an array.
+ *
+ * @param {Array} array
+ * @param {String} element
+ * @returns {number}
+ */
+function findIndexOfElementInArray(array, element) {
+  return _findIndex(array, (item) => item[element]);
```

Comment:
I think we can afford this one by just using vanilla JS

```suggestion
  return array.findIndex((item) => Boolean(item[element]));
```

Reply:
+1